lib/src/core/better_player_with_controls.dart

```diff
@@ -42,6 +42,7 @@ class _BetterPlayerWithControlsState extends State<BetterPlayerWithControls> {
     playerVisibilityStreamController.add(true);
     _controllerEventSubscription =
         widget.controller!.controllerEventStream.listen(_onControllerChanged);
+    widget.controller!.setTrack(trackWithMaxBitrate());
```

Comment:
wydaje mi sie ze wybor tracka jest juz gdzieś tutaj, wiec tak defacto set track jest wołąny dwa razy. Proszę zwerfyikuj to, bo jeśli tak jest to ta zmiana nie poprawi performance, wrecz odwrotnie